platform/hosting/enterprise-licenses.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: Enterprise Licenses
+description: Learn about W&B Enterprise licenses, what features they include, and how to obtain and configure them.
+---
+
+An Enterprise license unlocks advanced W&B features designed for organizations that need enhanced capabilities in areas of security, compliance, or administrative. See the list of [Enterprise features](#enterprise-features).
+
+- **Security**: Enhanced authentication, encryption, and access controls.
+- **Compliance**: Audit logging, HIPAA compliance options, and data governance controls.
+- **Administration**: Advanced user management features like SCIM provisioning, custom roles, and automation capabilities.
+
+This page provides a comprehensive overview of Enterprise licenses, including what features they enable, how to obtain them, and how to configure them for your deployment.
+
+## Deployment Options
+Enterprise licenses are available for W&B Dedicated Cloud and Self-Managed deployments.
+
+- [W&B Dedicated Cloud](/platform/hosting/hosting-options/dedicated_cloud) provides single-tenant infrastructure managed by W&B, deployed in Google Cloud. Dedicated Cloud deployments automatically include an Enterprise license. No additional configuration is required, and all Enterprise features are available immediately upon provisioning.
+- [W&B Self-Managed](/platform/hosting/hosting-options/self-managed) is deployed on your own infrastructure, providing full control over your deployment and data. Air-gapped deployments are supported. To access Enterprise features on W&B Self-Managed, you must obtain and configure an Enterprise license. Without an Enterprise license, Self-Managed deployments operate with limited features.
+
+## Obtain an Enterprise license
+This section explains how to obtain an Enterprise license. Select a tab to continue.
+
+<Tabs>
+<Tab title="Dedicated Cloud">
+W&B Dedicated Cloud includes an Enterprise license and no action is required. To get started with Dedicated Cloud:
+
+1. Contact [Sales](https://wandb.ai/site/contact-sales) to discuss your requirements.
+2. Work with W&B to provision your dedicated instance.
+3. All Enterprise features will be enabled upon deployment.
```

Comment:
If this info is covered elsewhere, we can omit these steps and instead link to said doc.